packages/grid/src/Grid.tsx

```diff
@@ -1887,9 +1868,13 @@ class Grid extends PureComponent<GridProps, GridState> {
    * @param value New value set
    */
   handleEditCellChange(value: string): void {
-    this.setState(({ editingCell }) => ({
-      editingCell: { ...editingCell, value } as EditingCell,
-    }));
+    this.setState(({ editingCell }) => {
+      assertIsDefined(editingCell);
```

Comment:
Can you check that this doesn't break React if `editingCell` is undefined? Or just throw an error in here and see what happens. I have no idea what react does in that case, but I'm assuming it's ok and just reports the error without breaking everything else

Reply:
It breaks everything. I'll add a try/catch around it